nixos/tests/docker-containers.nix

```diff
@@ -1,20 +1,21 @@
 # Test Docker containers as systemd units
 
-import ./make-test.nix ({ pkgs, lib, ... }: {
+import ./make-test.nix ({ pkgs, lib, ... }:
+
+{
   name = "docker-containers";
   meta = {
-    maintainers = with lib.maintainers; [ benley ];
+    maintainers = with lib.maintainers; [ benley mkaito ];
   };
 
   nodes = {
     docker = { pkgs, ... }:
       {
         virtualisation.docker.enable = true;
 
-        virtualisation.dockerPreloader.images = [ pkgs.dockerTools.examples.nginx ];
-
         docker-containers.nginx = {
           image = "nginx-container";
+          imageFile = pkgs.dockerTools.examples.nginx
```

Comment:
There is a syntax error. Please at least run the test first.

Reply:
> There is a syntax error.

Fixed, my bad.

> Please at least run the test first.

Please at least stop being condescending to people who actually contribute, lest they stop contributing.

Reply:
thanks for your work :heart: